include/cling/Interpreter/Interpreter.h

```diff
@@ -613,6 +613,7 @@ namespace cling {
 
     clang::CompilerInstance* getCI() const;
     clang::Sema& getSema() const;
+    IncrementalParser& getIncrParser() const { return *m_IncrParser; }
```

Comment:
Lost in all the noise.
Is this the final word, and if so can the commit just be ignored on you side or must I chop it off?